core/src/main/java/org/frankframework/receivers/Receiver.java

```diff
@@ -210,7 +208,7 @@
 @FrankDocGroup(FrankDocGroupValue.OTHER)
 public class Receiver<M> extends TransactionAttributes implements ManagableLifecycle, IMessageHandler<M>, IProvidesMessageBrowsers<M>, EventThrowing, IbisExceptionListener, HasSender, FrankElement, IThreadCountControllable, NameAware {
 	private final @Getter ClassLoader configurationClassLoader = Thread.currentThread().getContextClassLoader();
-	private @Getter @Setter ApplicationContext applicationContext;
+	private @Getter ApplicationContext applicationContext;
```

Comment:
Is dit veld nodig? Kan je niet een getter maken die gewoon `return adapter` doet?